pkg/clusteragent/externalmetrics/model/datadogmetricinternal.go

```diff
@@ -98,6 +102,18 @@ func NewDatadogMetricInternal(id string, datadogMetric datadoghq.DatadogMetric)
 	return internal
 }
 
+func hasForceActiveAnnotation(metric datadoghq.DatadogMetric) bool {
+	if value, found := metric.Annotations[alwaysActiveAnnotation]; found {
+		enabled, err := strconv.ParseBool(value)
+		if err != nil {
+			log.Errorf("Unable to parse value from %s annotation: '%s'", alwaysActiveAnnotation, value)
```

Comment:
We usually don't log errors on user misconfigurations. Given the scope of the feature, we could log `Debug` or not log at all.

Reply:
ok !